data/json/monsters/mechsuits.json

```diff
@@ -0,0 +1,139 @@
+[
+  {
+    "id": "mon_mech_recon",
+    "type": "MONSTER",
+    "name": "X-03: 'Spectre' recon mech",
+    "description": "The Boeing-Daewoo RMES ( Recon Mechanical Exoskeleton Suit ), a recent acquisition by the US military, it was designed to be used in a scout-recon-sniper role, due to its mobility and integrated laser sniper rifle and suite of optics for target designation and battlefield awareness.  It was not deployed before the Cataclsym hit, though there were a few prototypes in the field.  You may be able to hack it to accept you as its pilot.  Like all mech-suits it can act as a UPS from its large battery.",
+    "default_faction": "mech_bot",
+    "species": [ "ROBOT" ],
+    "diff": 5,
+    "volume": "700388 ml",
+    "weight": 700388,
+    "hp": 320,
+    "speed": 120,
+    "material": [ "superalloy" ],
+    "symbol": "M",
+    "color": "blue",
+    "aggression": -50,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 3,
+    "melee_dice_sides": 8,
+    "melee_cut": 5,
+    "armor_bash": 15,
+    "armor_cut": 25,
+    "vision_night": 100,
+    "mech_str_bonus": 5,
+    "mech_battery": "huge_atomic_battery_cell",
+    "mech_weapon": "recon_mech_laser",
+    "revert_to_itype": "broken_mech_recon",
+    "death_function": [ "BROKEN" ],
+    "death_drops": { "groups": [ [ "robots", 20 ] ] },
+    "flags": [
+      "SEES",
+      "MECH_RECON_VISION",
+      "HEARS",
+      "MILITARY_MECH",
+      "BASHES",
+      "PUSH_MON",
+      "ELECTRONIC",
+      "NOHEAD",
+      "NO_BREATHE",
+      "RIDEABLE_MECH",
+      "PRIORITIZE_TARGETS",
+      "PATH_AVOID_DANGER_2"
+    ]
+  },
+  {
+    "id": "mon_mech_combat",
+    "type": "MONSTER",
+    "name": "X-02: 'Grunt' Combat Mech",
+    "description": "The Boeing-Daewoo CMES ( Combat Mechanical Exoskeleton Suit ), a recent acquisition by the US military, it was designed to be used in a fire support role, due to its fearsome integrated gatling laser.  It was not deployed before the Cataclsym hit, though there were a few prototypes in the field.  You may be able to hack it to accept you as its pilot.  Like all mech-suits it can act as a UPS from its large battery.",
```

Comment:
```suggestion
    "description": "The Boeing-Daewoo CMES (Combat Mechanical Exoskeleton Suit), a recent acquisition by the US military, it was designed to be used in a fire support role, due to its fearsome integrated gatling laser.  It was not deployed before the Cataclysm hit, though there were a few prototypes in the field.  You may be able to hack it to accept you as its pilot.  Like all mech-suits it can act as a UPS from its large battery.",
```